### ISSUE TITLE
Support for python 3.6

### ISSUE THREAD

**Issue by postlund** [postlund/pyatv#18]
There seems to be a bug in pylint/asteriod that causes a [circle dependency](https://travis-ci.org/postlund/pyatv/jobs/198384154):

```
...
  File "/home/travis/build/postlund/pyatv/.tox/lint/lib/python3.6/site-packages/astroid/rebuilder.py", line 167, in visit
    return visit_method(node, parent, assign_ctx)
  File "/home/travis/build/postlund/pyatv/.tox/lint/lib/python3.6/site-packages/astroid/rebuilder.py", line 447, in visit_expr
    newnode.value = self.visit(node.value, newnode, assign_ctx)
  File "/home/travis/build/postlund/pyatv/.tox/lint/lib/python3.6/site-packages/astroid/rebuilder.py", line 167, in visit
    return visit_method(node, parent, assign_ctx)
  File "/home/travis/build/postlund/pyatv/.tox/lint/lib/python3.6/site-packages/astroid/rebuilder.py", line 315, in visit_call
    for child in node.args]
  File "/home/travis/build/postlund/pyatv/.tox/lint/lib/python3.6/site-packages/astroid/rebuilder.py", line 315, in <listcomp>
    for child in node.args]
  File "/home/travis/build/postlund/pyatv/.tox/lint/lib/python3.6/site-packages/astroid/rebuilder.py", line 165, in visit
    visit_method = getattr(self, visit_name)
AttributeError: 'TreeRebuilder3k' object has no attribute 'visit_joinedstr'
ERROR: InvocationError: '/home/travis/build/postlund/pyatv/.tox/lint/bin/pylint pyatv'
lint runtests: commands[2] | pydocstyle pyatv tests
```

From what I can see in https://github.com/PyCQA/pylint/issues/1113, this will be fixed when PyLint 2.0 and asteroid 1.5 is released. So I will let PR #15 wait until that it fixed.

**Comment by nriley** [comment]
I have been getting more unexpected errors in Python 3.5 than 3.6 at this point — perhaps this is out of date?

**Comment by postlund** [comment]
My assumption is that it works just fine with python 3.6, but current state of pylint makes test fail (on Travis in this case). Until I can automatically run tests I consider that it's not officially supported.

**Comment by postlund** [comment]
Fixed by #15 with latest versions of various test software 👍
